libraries/hatch-server-user-management/src/UserManagementController.ts

```diff
@@ -14,179 +14,243 @@ import {
   StartPasswordResetRequest,
   ConfirmPasswordResetRequest,
   SignOutUserRequest,
-  ResendUserRegistrationCodeRequest
+  ResendUserRegistrationCodeRequest,
+  SetUserAttributesRequest,
 } from './UserManagementRequests';
 import UserContext from './UserContext';
+import * as HttpStatus from 'http-status-codes'
 
 export const AUTH_WHITELIST_KEY = 'AUTH_WHITELIST_KEY';
 
 @controller()
 export default class UserManagementController {
   
-  private authWhitelist = [
+  private readonly authWhitelist = [
     // default whitelist for swagger
     '/favicon.ico',
     '/api',
     '/api.json',
   ];
   
   constructor(
-    @inject('UserServiceClient') private readonly userService: UserServiceClient,
+    @inject('UserManagementClient') private readonly userService: UserManagementClient,
     @inject('Logger') private readonly logger: Logger,
     @injectAll(AUTH_WHITELIST_KEY) customAuthWhitelist: string[],
   ) {
     if (customAuthWhitelist.length > 0) {
       this.authWhitelist = this.authWhitelist.concat(customAuthWhitelist);
     }
-    this.logger.debug('Auth whitelist:', this.authWhitelist);
+    this.logger.debug('Auth whitelist: ', this.authWhitelist);
```

Comment:
This was intentional -- doesn't the comma result in a space?

Reply:
you are correct, updated